protocol.md

```diff
@@ -114,6 +114,114 @@ Observe the behavior of this algorithm in some failure scenarios. First, any of
+
+At the end of the KYC authentication process, the KYC company would send Bob a confirmation that Alice is actually who she claims to be.  It would include a unique identifier, so that if Alice is KYCed multiple times, Bob will know that it is the same person each time. This is the information Bob will use to connect Alice\'s multiple secrets, if she pairs multiple times.
+
+A sharer and helper can agree ahead of time for how they want to handle authentication and recovery. If the authentication is in person, then recovery could be done as in the previous section. If the authentication is over the internet, then the agreement might be that Alice will first KYC normally, and then the KYC company will spend a full week trying to contact Alice at all know addresses (email, text, postal address), to confirm that she is actually doing the recovery, and is not being attacked by an imposter trying to connect as her. 
+
+The agreement might also contain other provisions, such as saying that if Alice is incapacitated, then her heirs are allowed to authenticate and recover her secrets only after meeting certain conditions, such as court order certifying that she is incapacitated or deceased. 
+
+Any agreements about authentication and recovery are outside the protocol. They are between the users. The DeRec Alliance may publish material about suggested best practices, and might even curate a collection of legal documents that might be used.
+
+### Business authentication
+
+It is possible that a company might provide helper as a service, and do their own KYC, rather than relying on a third party. For example, a bank or credit union may already have procedures in place for KYC and authentication of customers. In this case, authentication, pairing, and recovery would be handled exactly as in the previous section, except that the helper company and the KYC company are actually the same company. A credit union or bank might charge for their helper service. Or they might provide it as a free service to their customers, to help attract and retain customers.
+
+## DeRec Concepts
+
+These are some of the major concepts related to DeRec, collected here in a list, for easy reference.
+
+- Sharer 
+	- An app that is protecting a secret by splitting shares of it among multiple helpers
+- Helper
+	- An app that can hold shares for one or more sharers
+	- A single app might act as both a sharer and helper, which means running two instances of the protocol in parallel
+- Secret
+	- The string of bytes that are to be protected. 
```

Comment:
You can see the definitions in both the protobuf repo and api-java repo. The syntax is different for protobuf vs Java. But in both cases, the definition means array of bytes. (Not unsigned bytes, because Java doesn't have that data type; UGH!).